python-sdk/src/astro/databases/base.py

```diff
@@ -68,9 +68,12 @@ class BaseDatabase(ABC):
     NATIVE_AUTODETECT_SCHEMA_CONFIG: Mapping[FileLocation, Mapping[str, list[FileType] | Callable]] = {}
     FILE_PATTERN_BASED_AUTODETECT_SCHEMA_SUPPORTED: set[FileLocation] = set()
 
-    def __init__(self, conn_id: str, table: BaseTable | None = None):  # skipcq: PYL-W0613
+    def __init__(
+        self, conn_id: str, table: BaseTable | None = None, load_options: LoadOptions | None = None
```

Comment:
Does it make sense to make load_options a database level member? Is there a scenario where we'd want to load with different options using the same database? cc: @tatiana @kaxil

Reply:
So far we are not using multiple loading options per DB/File, I think it makes sense to put them at the initialization step. Anyway, we can still cater to the scenario of using different load options by providing helper methods to update those on the fly.

Reply:
Ok I think I'm ok with this since users never actually create database objects.